x-pack/plugins/ml/public/application/datavisualizer/file_based/components/fields_stats_grid/fields_stats_grid.tsx

```diff
@@ -0,0 +1,202 @@
+            gutterSize="s"
+            alignItems="center"
+            style={{ maxWidth: 250 }}
+            data-test-subj="mlDataVisualizerFieldsSummary"
+          >
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <h5>
+                  <FormattedMessage
+                    id="xpack.ml.dataVisualizer.searchPanel.allFieldsLabel"
+                    defaultMessage="All fields"
+                  />
+                </h5>
+              </EuiText>
+            </EuiFlexItem>
+
+            <EuiFlexItem grow={false}>
+              <EuiNotificationBadge
+                color="subdued"
+                size="m"
+                data-test-subj="mlDataVisualizerVisibleFieldsCount"
+              >
+                <strong>{fieldsCountStats.visibleFieldsCount}</strong>
+              </EuiNotificationBadge>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText color="subdued" size="s" data-test-subj="mlDataVisualizerTotalFieldsCount">
+                <FormattedMessage
+                  id="xpack.ml.dataVisualizer.searchPanel.ofFieldsTotal"
+                  defaultMessage="of {totalCount} total"
+                  values={{ totalCount: fieldsCountStats.totalFieldsCount }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        )}
+
+        {metricsStats && (
+          <EuiFlexGroup
+            gutterSize="s"
+            alignItems="center"
+            style={{ maxWidth: 250 }}
+            data-test-subj="mlDataVisualizerMetricFieldsSummary"
+          >
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <h5>
+                  <FormattedMessage
+                    id="xpack.ml.dataVisualizer.searchPanel.numberFieldsLabel"
+                    defaultMessage="Number fields"
+                  />
+                </h5>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiNotificationBadge
+                color="subdued"
+                size="m"
+                data-test-subj="mlDataVisualizerVisibleMetricFieldsCount"
+              >
+                <strong>{metricsStats.visibleMetricsCount}</strong>
+              </EuiNotificationBadge>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText color="subdued" size="s" data-test-subj="mlDataVisualizerMetricFieldsCount">
+                <FormattedMessage
+                  id="xpack.ml.dataVisualizer.searchPanel.ofFieldsTotal"
+                  defaultMessage="of {totalCount} total"
+                  values={{ totalCount: metricsStats.totalMetricFieldsCount }}
+                />
+              </EuiText>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        )}
+
+        <EuiFlexItem style={{ alignItems: 'flex-end' }}>
+          <EuiFlexGroup gutterSize="xs" data-test-subj="mlDataVisualizerFieldCountPanel">
```

Comment:
I think you can use the `alignItems` prop in `EuiFlexGroup ` instead?

Reply:
You're right. Updated here https://github.com/elastic/kibana/pull/87598/commits/a640650c72b5d0289aa0842622fa9b28fdcb1828.